db/changes.go

```diff
@@ -680,6 +680,12 @@ func (db *Database) SimpleMultiChangesFeed(chans base.Set, options ChangesOption
 			base.DebugfCtx(db.Ctx, base.KeyChanges, "MultiChangesFeed waiting... %s", base.UD(to))
 			output <- nil
 
+			// If this is an initial replication (from zero) with the activeOnly set, flip it now the client has caught up.
+			if !options.Since.IsNonZero() && options.ActiveOnly {
```

Comment:
If the initial iteration of the loop sent any data, options.Since will have been updated for use in the second iteration, and will no longer be zero.  I think you'll need to set a 'fromZero' flag at the start of SimpleMultiChanges feed, and check it here.

Reply:
good catch - I've rearranged my test to only start the client replicator after the initial doc creation to catch this.